app/controllers/application_controller.rb

```diff
@@ -17,7 +17,7 @@ class ApplicationController < ActionController::Base
     Rails.logger.warn(e.message)
 
     if rendering_context == "modal"
-      render nothing: true, status: :bad_request
+      head :bad_request
```

Comment:
Oh good catch - I feel like we might want to instead do `raise ActionController::BadRequest`

Reply:
I think we should keep this as-is, so that both branches of the conditional have a similar level of abstraction. Another reason I'd like to keep it is that we're already doing some handling by logging a warning and checking the context, so I'd expect us to do all the handling here. Is there a counter-argument here?

Reply:
The counter argument would be that it's a consistent way to handle bad requests across the app when we're not concerned about a particular view. If anyone happens to get this response rendered to their browser they'll just get a blank screen.

Reply:
True. I thought we were being deliberately inconsistent here, given the modal context, but that's not the case. Updated.